augly/audio/functional.py

```diff
@@ -67,7 +67,8 @@ def add_background_noise(
     if background_audio is None:
         background_audio = random_generator.standard_normal(audio.shape)
     else:
-        background_audio, _ = audutils.validate_and_load_audio(background_audio, 1)
+        background_audio, _ = audutils.validate_and_load_audio(
```

Comment:
Please get rid of these formatting changes which are unrelated to your PR :) It makes it harder to see your actual changes; we know we need to reformat all files using `black`, we will do it soon in a separate PR.